cocli/README.md

```diff
@@ -437,6 +437,16 @@ $ cocli corim submit \
 >> "corim.cbor" submit ok
 ```
 
+#### Remote Service Authentication
+
+The above will work if the remote service does not authenticate
```

Comment:
Just a question here, more than a comment.

Does the `cocli` configuration, in any way needs to be synchronised with the 
equivalent configuration on the services and apiclient side?
Specifically my question originates from the following line 11 on the config file:
 ( auth: none)

Reply:
Also it would be good to specify what is the default `cocli` behaviour!

Reply:
This configuration is used to configure the `apiclient` so no synchronisation needed, as such, for that. The configuration for cocli does need to match the corresponding configuration for the service it is contacting.

The default is set in the example config, and is specified in the help for the corresponding flag (when you do `cocli corim submit  -h`).